int/hwsc-file-transaction-svc/hwsc-file-transaction-svc.js

```diff
@@ -13,7 +13,7 @@ const hwscFileTransactionSvcProtoPkgDef = protoLoader.loadSync('', options);
 const hwscFileTransactionSvcPbJs = grpc.loadPackageDefinition(hwscFileTransactionSvcProtoPkgDef)
   .hwscFileTransactionSvc;
 
-const client = new hwscFileTransactionSvcPbJs.FileTransactionService('localhost:50051',
+const client = new hwscFileTransactionSvcPbJs.FileTransactionService('localhost:50055',
```

Comment:
Is this the only change? Because I only see this.

Reply:
yup this is the only change